orchestrator-salt/src/main/resources/salt-common/salt/monitoring/template/prometheus.yml.j2

```diff
@@ -9,6 +9,12 @@
   {%- if instance_cmd_output %}
     {%- set instance_name = instance_cmd_output %}
   {%- endif %}
+  {%- if salt['pkg.version_cmp'](telemetry.cdpTelemetryPackageVersion,'0.4.29-1') >= 0 %}
+    {%- set get_doctor_addresses = "cdp-doctor network addresses --format json --cloud-provider {} --databus-url {} --databus-s3-url {}".format(telemetry.platform, telemetry.databusEndpoint, telemetry.databusS3Endpoint) %}
+    {%- set doctorNetworkAddresses = salt.cmd.run(get_doctor_addresses) | load_json %}
+    {%- set blackboxInternalUrls = doctorNetworkAddresses['cloudera'] %}
```

Comment:
i was just wondering this should be the external as cloudera addresses are outside of the cloud context of the VMs...but to be fair it could be a bit complicated, maybe we should make a difference based on cloud specific and cloudera specific addresses, so could be internal/external name was not a good idea, so could be like cloudProviderUrls and clouderaUrls ?

Reply:
ok